### PR TITLE
update installing_cloudforms_on_red_hat_enterprise_virtualization deleting downstream references

### PR DESCRIPTION
**Pull request by katafira**
*************************************************************************************************************
Issue: 
	There are references to various .adoc files, but with the names of said files changed to lower case there might be conflicts. What are we supposed to do?

*************************************************************************************************************

************************************* file: configuring_cloudform.adoc ************************************
line: 73 rephrased from:

======
Before installing an internal database, add a disk to the infrastructure hosting your appliance. See the documentation specific to your infrastructure for instructions on how to add a disk. As a storage disk usually cannot be added while a virtual machine is running, Red Hat recommends adding the disk before starting the appliance. {product-title} only supports installing of an internal VMDB on blank disks. The installation will fail if the disks are not blank.
======

To:

======
Before installing an internal database, add a disk to the infrastructure hosting your appliance. See the documentation specific to your infrastructure for instructions on how to add a disk. As a storage disk usually cannot be added while a virtual machine is running, adding the disk before starting the appliance is recommended. {product-title} only supports installing of an internal VMDB on blank disks. The installation will fail if the disks are not blank.
======

Rephrased line 108 form:

The `postgresql.conf` file used with {product-title} databases requires specific settings for correct operation. For example, it must correctly reclaim table space, control session timeouts, and format the PostgreSQL server log for improved system support. Due to these requirements, Red Hat recommends that external {product-title} databases use a `postgresql.conf` file based on the standard file used by the {product-title} appliance.

To:

The `postgresql.conf` file used with {product-title} databases requires specific settings for correct operation. For example, it must correctly reclaim table space, control session timeouts, and format the PostgreSQL server log for improved system support. Due to these requirements, it is recommended that external {product-title} databases use a `postgresql.conf` file based on the standard file used by the {product-title} appliance.
**********************************************************************************************************

********************************** file: installing_cloudforms.adoc **************************************
Issue on line 17:

. From the list of installers and images, select the *CFME Red Hat Virtual Appliance* download link.

!!!!! We do not know if any changes are needed there !!!!!!!!!!

Issue on lines: 19, 21, 36, 107, 111, 115, 162, 171, 174

!!! We do not know if any changes are needed in "Red Hat Enterprise Virtualization Manager/Red Hat Enterprise Virtualization Management" !!!

**********************************************************************************************************

### CODE REVIEW

File: installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+======
+Before installing an internal database, add a disk to the infrastructure hosting your appliance. See the documentation specific to your infrastructure for instructions on how to add a disk. As a storage disk usually cannot be added while a virtual machine is running, Red Hat recommends adding the disk before starting the appliance. {product-title} only supports installing of an internal VMDB on blank disks. The installation will fail if the disks are not blank.
+======
+
+
+To:
+
+======
+Before installing an internal database, add a disk to the infrastructure hosting your appliance. See the documentation specific to your infrastructure for instructions on how to add a disk. As a storage disk usually cannot be added while a virtual machine is running, adding the disk before starting the appliance is recommended. {product-title} only supports installing of an internal VMDB on blank disks. The installation will fail if the disks are not blank.
+======
```
Comment (elladeon) [inline installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc]: This looks good to me.

File: installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Rephrased line 108 form:
+
+The `postgresql.conf` file used with {product-title} databases requires specific settings for correct operation. For example, it must correctly reclaim table space, control session timeouts, and format the PostgreSQL server log for improved system support. Due to these requirements, Red Hat recommends that external {product-title} databases use a `postgresql.conf` file based on the standard file used by the {product-title} appliance.
+
+
+To:
+
+The `postgresql.conf` file used with {product-title} databases requires specific settings for correct operation. For example, it must correctly reclaim table space, control session timeouts, and format the PostgreSQL server log for improved system support. Due to these requirements, it is recommended that external {product-title} databases use a `postgresql.conf` file based on the standard file used by the {product-title} appliance.
+**********************************************************************************************************
+
```
Comment (elladeon) [inline installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc]: This looks good to me.

File: installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+. From the list of installers and images, select the *CFME Red Hat Virtual Appliance* download link.
+
+
+!!!!! We do not know if any changes are needed there !!!!!!!!!!
+
+
+Issue on lines: 19, 21, 36, 107, 111, 115, 162, 171, 174
+
+!!! We do not know if any changes are needed in "Red Hat Enterprise Virtualization Manager/Red Hat Enterprise Virtualization Management" !!!
+
```
Comment (elladeon) [inline installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc]: I'll kick it to @Fryguy, but I believe the RHEV references can stay as they are. There is a ManageIQ appliance for RHEV as well as oVirt.

File: installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+The `postgresql.conf` file used with {product-title} databases requires specific settings for correct operation. For example, it must correctly reclaim table space, control session timeouts, and format the PostgreSQL server log for improved system support. Due to these requirements, it is recommended that external {product-title} databases use a `postgresql.conf` file based on the standard file used by the {product-title} appliance.
+**********************************************************************************************************
+
+
+********************************** file: installing_cloudforms.adoc **************************************
+Issue on line 17:
+
+. From the list of installers and images, select the *CFME Red Hat Virtual Appliance* download link.
+
```
Comment (elladeon) [inline installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc]: The drop down menu is here: http://manageiq.org/download/

You can choose RHEV from that list.

However, isn't RHEV already done?

Comment (katafira) [inline installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc]: What do you mean by RHEV being already done? We were sent a .doc with a table with needed changes in the documentation. Are you talking about the documentation or the development project?

Comment (sergio-ocon) [inline installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc]: @katafira thtat is the address to download the manageiq ova to install in rhev.

**Comment by chessbyte** [comment]
@katafira what is the status of this PR?

**Comment by katafira** [comment]
Removed PR 24, the good one is #54

**Pull request closed without merging**